include/mfmg/cuda/dealii_operator_device_helpers.cuh

```diff
@@ -40,7 +40,7 @@ void lu_factorization(cusolverSpHandle_t cusolver_sp_handle,
                       SparseMatrixDevice<ScalarType> const &matrix,
                       ScalarType const *b, ScalarType *x);
 
-__global__ void iota(int const size, int *value);
+__global__ void iota(int const size, int *value, int const offset = 0);
```

Comment:
I find the naming of the variables disturbing:
* `offset` usually refers to a distance between memory location.
* in all the documentations of `std::iota` I have seen, the last argument that specifies the initial value is always called `val` or `value`.

I suggest you rename:
* `value` -> `first` or `data`
* `offset` -> `value`